PersonsApi/appsettings.Development.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Debug",
```

Comment:
Default log level should be 'warning' or greater and switched to debug only when necessary to avoid excess logging